src/dsp_tools/cli.py

```diff
@@ -449,7 +449,7 @@ def _call_requested_action(args: argparse.Namespace) -> bool:
                     password=args.password,
                     dump=args.dump,
                 )
-        else:
+        else:  # noqa: PLR5501 (collapsible-else-if)
```

Comment:
I'm with ruff on this one

Reply:
I'm happy to agree to disagree on this one :)  
but generally I think it's not a wise idea to employ an opinionated tool and not agree with the opinions it has. So I would vote for not making that the rule but the exception